app/models/restaurant.rb

```diff
@@ -32,8 +29,25 @@ def calculate_rating
       ratings += r.rating
     end
     ratings / reviews.count
+  end
 
+  def parse_csv(file_name)
```

Comment:
You can place the loading task in `tasks/load_restaurants.rake`

Since this logic isnt related to the model itself of the restaurant but more related to the general logic in the app.

Reply:
do i need to open a new directory called tasks?
or the tasks directory is exist and i just missed it?